conf.d/python.d/apache_cache.conf

```diff
@@ -0,0 +1,4 @@
+# Example configuration of apache_cache.chart.py
+# YAML format
+
+path: "/var/log/apache2/cache.log"
```

Comment:
remember that if you write a name it will be shown in netdata dashboard. Thus configuration like this:
```yaml
local:
  path: "/var/log/apache2/cache.log"
```
is valid, but will be show in netdata as `apache_cache_local` instead of simple `apache_cache`